data/localized/de_patch/text/conversations/11_hearthsong/11_cv_gilded_vale_villager.stringtable

```diff
@@ -41,7 +41,7 @@
     </Entry>
     <Entry>
       <ID>8</ID>
-      <DefaultText>„Wir dachten, wir hätten die Burg eine Weile für uns allein, nachdem du weg warst. Bis sie jemand neuen schicken. Aber eines Nachts hören wir etwas im Thronsaal und&#160;… und finden ihn. Fürst Raedric.“</DefaultText>
+      <DefaultText>„Wir dachten, wir hätten die Burg eine Weile für uns allein, nachdem du weg gewesen warst. Bis sie jemand neuen schicken. Aber eines Nachts hören wir etwas im Thronsaal und&#160;… und finden ihn. Fürst Raedric.“</DefaultText>
```

Comment:
Wenn der Satz einigermaßen Präteritum sein soll, dann müssten wir konsequenterweise auch „fanden“ schreiben.

Reply:
… und fanden ihn: Fürst Raedric.

Ich finde da wird ein Doppelpunkt gesprochen, mindestens ein – wenn Doppelpunkte nicht überzeugen.

Reply:
Müßte dann auch nicht ein "hörten" rein? Wobei es schon komisch im englischen Text ist.

"Until they sent someone new. But one night we hear something in the throne room, and... and it's him. Lord Raedric."